[pull] primary from flybywiresim:primary (#54)
Valastiri/fbw-userguide: docs/beginner-guide/after-landing-steps.md

### Lower Pedestal

The following actions should be preformed on the lower pedestal

=== "Left Side Lower Pedestal"

>     * Set your radio frequency to the one assigned by atc if using atc

Reviewer: ```suggestion
  * Set your radio frequency to the one assigned by ATC
    * ATC is set in accordance with airport requirements.
```

    * Disarm the spoilers by pushing down on the lever

    * ENG mode selector to norm

        ![Bottom left](../beginner-guide/placeholder.jpg)

=== "Right Side Lower Pedestal"

    * Retract your flaps to zero

    * Set your tcas to standby

    * Turn your radar off

    * Turn your predictive windshear system off

        ![Bottom right positions](../beginner-guide/placeholder.jpg)

### Overhead Panel

The following actions should be preformed on the overhead panel and top ecam

=== "Left Side Overhead Panel"

    Set your exterior lights

>     !!! info "Exterior Lights"

Reviewer: The info block is probably not needed here. You haven't used it anywhere else and the bullet points should suffice.

        * Set your noise wheel light to taxi

        * Nav on

        * Beacon on

        * Runway turn-offs on

        * Turn your strobe lights off unless you are crossing an active runway

        * For landing lights, you can choose to retract them or turn them off

        Your landing lights should look like this:

        ![landing light positions](../beginner-guide/placeholder.jpg)

=== "Right Side Overhead Panel"

    * Start your APU by clicking the APU master button then the APU start button

>     * Turn your ati-ice off if not needed

Reviewer: ```suggestion
    * Turn your ati-ice off if not needed
    * Turn off your anti-ice if it is not required
```

    * Turn the brake fan on if you get a hot brakes ecam warning

    ![Right Side Overhead Panel positions](../beginner-guide/placeholder.jpg)

> === "Turning into gate"

Reviewer: ```suggestion
=== "Turning into gate"
=== "Turning Into Gate"
```

    * When turning into the gate turn off your nose wheel light and your runway turn off lights

    ![Right Side Overhead Panel](../beginner-guide/placeholder.jpg)

>     * Continue with the [Powering Down](TBD.MD) steps

Reviewer 2: Who ever makes the parking page you will need to change this the TBD.MD to the file name

Reviewer: This is actually not needed. The mkdocs has a next page function built in.

Reviewer 2: Ok

Reviewer: Can you do `[Powering Down](#)` just so there's no strange build errors

Reviewer: See updated review

>     Continue with the [Powering Down](TBD.MD) steps

Reviewer: ```suggestion
    Continue with the [Powering Down](#) steps
```
